### PR TITLE
Iteration1

### PR DESCRIPTION
**Pull request by SullyBirashk**
Creates StatTracker class that can open CSV files.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,11 +1,24 @@
 require "csv"
 
 class StatTracker
+  attr_reader :game, :teams, :game_teams, :file_paths, :file_hash
+
+  def initialize(file_hash)
+    file_paths = StatTracker.create_csv_hash(file_hash)
+    @games = file_paths[:games]
```
Comment (SullyBirashk) [inline lib/stat_tracker.rb:8]: I appreciate the fact that this saves us a step by creating the file paths in the initialize!

Comment (jake-edel) [reply lib/stat_tracker.rb:8]: yeah, seemed like a logical move

**Review by CooleyNick — COMMENTED** [review]
This is going to get iteration 2 off to a great start. It's easy to read and it helps me understand what we're trying to go for.

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+  end
+
   it 'exists' do
-    stat_tracker = StatTracker.new
-    expect(stat_tracker).to be_instance_of StatTracker
+    expect(@stat_tracker).to be_instance_of StatTracker
+  end
+
+  it "creates csv array" do
+    csv_array = StatTracker.open_csv_file_paths(@locations)
```
Comment (jake-edel) [inline spec/stat_tracker_spec.rb]: #open_csv_file_paths takes a Hash as an argument and returns an Array of CSV objects

**Review by jake-edel — COMMENTED** [review]
See comment on line 23

**Pull request merged**